### ISSUE TITLE
Do you have any plans to upload compressai code & pretrained model?

### ISSUE THREAD

**Issue by sybahk** [kamisli-icpl/Learned-block-based-image-compression#1]
Hello, I'm undergraduate student studying block-based learned image compression.  
First of all, I appreciate your contribution to block-based learned image compression, especially about fusing image blocks using masked convolution and ACL training.  
By the way, I tried to reproduce your result with compressai library, but Rate-Distortion curve of reproduced model didn't go higher than JPEG2000..  
I cross-checked model several times, but I couldn't find any difference with described model.  
Could you please upload your code and pretrained model in public? I will really appreciate that.  
Thank you!

**Comment by kamisli** [comment]
Dear Suyong Bahk, 
the paper is currently under review and I will publish the model and codes here once the review process proceeds. Sorry that you couldn't reproduce the results yourself.
Best,
-fatih
